Miscellaneous/scf-dictionary

```diff
@@ -0,0 +1,49 @@
+import pickle
+
+from wrap_pyscf import *
+
+scf_guess = {}
+
+filename = "scf_record"
+
+#with open(filename, 'rb') as outfile:
+    #scf_guess = pickle.load(outfile)
+
+if "EthaneA 1Cx2" not in scf_guess:
+    funct_grps = [[0,2,3,4], [1,5,6,7]]
+    scf_guess["EthaneA 1Cx2"] = guess_driver("EthaneA.sdf", funct_grps, [1,1])
+        
+if "PropaneA 1Cx3" not in scf_guess:
+    funct_gprs = [[2,0,1,3], [4,5,6], [7,8,9,10]]
+    scf_guess["PropaneA 1Cx3"] = guess_driver("PropaneA.sdf", funct_grps, [1,2,1])
+        
+if "PropaneA cc-c" not in scf_guess:
+    funct_grps = [[1,2,3,4,0,6,7], [5,8,9,10]]
```

Comment:
This fragmentation is wrong. 5 and 7 need to be in the opposite groups.

Reply:
I went in and obtained new coordinates in avogadro, and applied those changes to propaneA c-c-c. I neglected cc-c. Thanks.